test/memory.test.js

```diff
@@ -430,6 +431,17 @@ describe('Memory connector', function() {
       });
     });
 
+    it('should search array for value', function(done) {
+      User.find({where: {favoriteNumbers: 1}}, function(err, users) {
```

Comment:
BTW, it is not very straightforward to implements `contains` for SQL based connectors as we store the array value as stringified string in a column.